packages/route/tests/Functional/Infrastructure/Doctrine/EventListener/RouteChangedUpdaterTest.php

```diff
@@ -53,6 +53,7 @@ public function testUpdateRoute(
         array $routes,
         string $changeRoute,
         array $expectedRoutes,
+        int $expectedChangedRoutes,
```

Comment:
introduced a bug which skipped the `expectedHistoryRoutes` part. with additional assertCount this can now not longer happen.